Lib/test/test_concurrent_futures.py

```diff
@@ -591,6 +674,34 @@ def test_traceback(self):
         self.assertIn('raise RuntimeError(123) # some comment',
                       f1.getvalue())
 
+    def test_ressources_gced_in_workers(self):
+        # Ensure that argument for a job are correctly gc-ed after the job
+        # is finished
+        obj = EventfulGCObj(self.ctx)
+        future = self.executor.submit(id, obj)
+        future.result()
+
+        assert obj.event.wait(timeout=1)
```

Comment:
By convention, we'd use `self.assertTrue` (also so that assertions are still checked if running with `-O`).